lib/key.rb

```diff
@@ -0,0 +1,22 @@
+class Key
+  attr_reader :key, :a_key, :b_key, :c_key, :d_key
+  def initialize
+    @key = key
```

Comment:
whole key generated from random number with leading 0. Following are 2 digit permutations of key.